pypeit/core/wavecal/defs.py

```diff
@@ -49,6 +49,8 @@ def lines():
     line_dict['HeI'] = 2**7
     line_dict['OH_R24000'] = 2**8
     line_dict['OH_triplespec'] = 2**9
+    line_dict['OH_XSHOOTER'] = 2**10
```

Comment:
Note that this will become `2**12` when GMOS is merged in.
This is an `arclines` thing that only I understand.  ;)

Reply:
I don't actually think these things are being used. But maybe you know.